node/packages/aws-lambda-sdk/instrument/index.js

```diff
@@ -33,7 +34,6 @@ const writeRequest = async (event, context) => {
   });
   const payloadBuffer = (serverlessSdk._lastRequestBuffer =
     requestResponseProto.RequestResponse.encode(payload).finish());
-  process._rawDebug(`SERVERLESS_TELEMETRY.R.${payloadBuffer.toString('base64')}`);
```

Comment:
Did you mean to remove these messages? I think we still want to continue printing the req/res messages even when dev mode is enabled 🤔

Reply:
Yes, by call of @ac360 and @Mmarzex we're expected to no longer write it (it was agreed yesterday in Jira)

Reply:
Ahh okay, I guess it will be good to get @Mmarzex's eyes on this as well 👍